### DISCUSSION TITLE
Locations descriprion missing

### DISCUSSION THREAD

**Discussion by Powerbook001** [teslamate-org/teslamate#2033] [General]
It happened lately that the "Location" description does not appear. 
You can still display it on the map if you open the "locations" in graphana, but in some of them, only a few, the description field is empty, and there is no way to edit it.
Is there a way to fix the issue?

Thank you all
<img width="583" alt="Screenshot 2021-10-05 at 08 36 57" src="https://user-images.githubusercontent.com/16048535/135972556-1dff5cfc-411e-4b68-b851-17c0e059dbf4.png">

**Comment by adriankumpf** [comment]
Could be related to this https://github.com/adriankumpf/teslamate/issues/1652

**Comment by Powerbook001** [comment]
I read it all, but did not find a solution apart using Geofencing.
Is it correct?

**Reply by Powerbook001** [reply]
I realized that since I made the last update, now all "drives" are correctly draw on the map, but unless I have a geofencing set up, there is a "blank" on the location.

Is there something I can do to fix the issue?

Thank you all.
